<launch>
<node pkg="zed_cpu_ros" type="zed_cpu_ros" name="zed_cpu_ros_node" output="screen">
	<param name="resolution" value="1"/>
	<param name="frame_rate" value="30"/>
	<param name="config_file_location" value="$(find zed_cpu_ros)/config/SN1267.conf"/>
	<param name="show_image" value="false"/>
</node>
</launch>
